examples/typescript/package-lock.json

```diff
@@ -83,7 +83,6 @@
     "dropbox": {
       "version": "file:../..",
       "requires": {
-        "@types/node-fetch": "^2.5.7",
```

Comment:
Why is this being removed?

Reply:
I'm not really sure, I dont remember making this change. I'll add it back